scRNA-seq/03-dimension_reduction_scRNA-seq.Rmd

```diff
@@ -20,63 +20,62 @@ downstream analyses.
 
 ## Set Up 
 
-For these analyses, we will need Seurat as well as the dplyr pipe. 
-
 ```{r}
-library(Seurat)
+# Load libraries
+library(ggplot2)
 
 # Magrittr pipe
 `%>%` <- dplyr::`%>%`
 ```
 
-We need to reload the Seurat object with the data we set up and normalized 
-previously. 
-
-```{r Path to Seurat}
-# Load in Seurat object
-seurat_norm <- readRDS(file.path("data", "seurat_object.RDS"))
+For this module, we are going to use the dataset that we processed part of in 
+`02-tag-based_pre-processing_scRNA-seq.md`
+We have already filtered and `scran` normalized this data for you (following 
+the same kinds of steps we did in `01-normalizing_scRNA-seq.Rmd`).
```

Comment:
```suggestion
the same steps we did in `01-normalizing_scRNA-seq.Rmd`).
```
How do the steps differ (asking why you included `kinds of`)?

Reply:
The cutoffs are different, but the steps are otherwise the same.

Reply:
Okay, I might say that here then.